code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -421,7 +421,7 @@
 
 /obj/item/weapon/storage/belt/sabre
 	name = "sabre sheath"
-	desc = "An ornate sheath designed to hold an officer's blade."
+	desc = "An ornate sheath designed to hold an officer's blade. Alt-Click to quickly draw the sheathed blade."

```

Comment:
This is basically only for the sabre, if I was making quickdraw a thing for multiple items i'd probably standardize it, also I don't really know how to get into it.